packages/web-components/fast-components/src/tabs/tabs.styles.ts

```diff
@@ -86,13 +76,12 @@ export const tabsStyles: (
             display: grid;
             grid-template-rows: auto;
             grid-template-columns: auto 1fr;
+            grid-row-gap: 8px;
```

Comment:
Same as above, should this be based off of density or designUnit?

Reply:
updated